fluentui_core/src/main/java/com/microsoft/fluentui/theme/FluentTheme.kt

```diff
@@ -18,16 +18,20 @@ internal val LocalThemeMode = compositionLocalOf { ThemeMode.Auto }
 
 @Composable
 fun FluentTheme(
-        globalTokens: GlobalTokens = GlobalTokens(),
-        aliasTokens: AliasTokens = AliasTokens(),
-        controlTokens: ControlTokens = ControlTokens(),
+        globalTokens: GlobalTokens? = null,
+        aliasTokens: AliasTokens? = null,
+        controlTokens: ControlTokens? = null,
         themeMode: ThemeMode = ThemeMode.Auto,
         content: @Composable () -> Unit
 ) {
+    val appGlobalToken by AppThemeController.observeGlobalToken(initial = GlobalTokens())
```

Comment:
For activity level theming, if before config change (rotation of screen) theme get changed then on config change, activity screen wont be able to retain it last applied theme. Check, if like individual control tokens for control, here globalTokens, aliasTokens & controlTokens can be made remember savable to provide easy retention behavior of these tokens in activity.